testbed/HIL_Firmware/PDU_HIL/PDU_HIL/include/digitalPot.h

```diff
@@ -0,0 +1,16 @@
+#ifndef DIGITAL_POT_H
+#define DIGITAL_POT_H
+ 
+//check data sheet for AD5260BRUZ200 to retreive values
```

Comment:
I don’t have enough context to understand what this means. Either remove or add more detail